lib/google_search_data_viewer/keyword/queries/keywords_query.ex

```diff
@@ -0,0 +1,18 @@
+defmodule GoogleSearchDataViewer.Keyword.Queries.KeywordsQuery do
+  import Ecto.Query, warn: false
+
+  alias GoogleSearchDataViewer.Keyword.Schemas.KeywordUpload
+
+  def list_for_user(user_id) do
+    KeywordUpload
+    |> where(user_id: ^user_id)
+    |> order_by(desc: :inserted_at)
+    |> select([:id, :user_id, :name, :status, :updated_at, :inserted_at])
```

Comment:
```suggestion
    |> select([:id, :user_id, :name, :status, :updated_at, :inserted_at])
```

Should we remove this?

Reply:
What about I move the `select` part to the Context to only select the `name`, `status`, and `inserted_at` ? I am not sure I want all fields, especially the HTML when I am just displaying the List.

Reply:
I think the number of fields is small (6 fields) for the KeywordUpload table. So i think that's fine to select it as default, we don't need to specify every field in `select([...])` like in this case.

Reply:
Yeah but as there's no pagination the KeywordUpload list can be 10s or 100s of rows all including the `html` field which I think is kind of large. Maybe it's best to explicitly exclude the `html` field in the context